include/rospack/rospack.h

```diff
@@ -152,6 +152,7 @@ class ROSPACK_DECL Rosstackage
     std::tr1::unordered_map<std::string, Stackage*> stackages_;
     Stackage* findWithRecrawl(const std::string& name);
     void log(const std::string& level, const std::string& msg, bool append_errno);
+    void clearStackages();
```

Comment:
That also occurred to me, to, but I didn't think that we were ensuring ABI compatibility (instead counting on the rebuild-everything-and-version-lock-the-debs approach on the farm).